scheduler/roles/slurm/vars/main.yml

```diff
@@ -40,6 +40,14 @@ slurm_login_packages:
 slurm_dbd_packages:
   RedHat:
     - slurm-slurmdbd
+db_packages:
+  RedHat:
+    common:
```

Comment:
add the python3-PyMySQL under the mysql and maridb keys. No need of the new "common" key.